Miscellaneous/HackerRank/MatrixScript.py

```diff
@@ -0,0 +1,19 @@
+first_multiple_input = input().rstrip().split()
```

Comment:
Add proper description of the problem in commented form along with the link of the hackerrank question.